README.md

```diff
@@ -101,13 +103,19 @@ HCL Language Support: https://plugins.jetbrains.com/plugin/7808-hashicorp-terraf
    
    There is an issue with Terraform not using the correct profile when more than one profile in the AWs config file
    
-6. Initialize Terraform (if not done so previously):
+6. Set the following Terraform environment variables on the local environment:
+
+    * TF_VAR_tdr_account_number=*[account number of the environment to update]*
+    * TF_VAR_tdr_environment=*[name of the TDR environment to update]* 
```

Comment:
Is this step necessary if we're using workspaces? I think you can have workspace-specific variables, but I might be misremembering.

Reply:
Not sure I undestand what you mean here? https://www.terraform.io/docs/configuration/variables.html

Reply:
Sorry, I misremembered how Terraform worked.

I was just wondering if there was a way to avoid setting a `TF_VAR_tdr_environment` variable, since we should be able to work out the environment from the workspace. Would it work if you used `terraform.workspace` instead of `var.tdr_environment`?

Reply:
Added as terraform.workspace so no need to set the TF_var